assets/src/conversion/index.js

```diff
@@ -19,6 +24,9 @@ class Conversion extends Component {
 			conversionBatchSize: '',
 			queuedBatchesCsv: '',
 			maxBatch: '',
+			hasConvertedPosts: '',
+			hasFailedConversions: '',
```

Comment:
Similar as in the other comment, these two booleans are oddly initiated as strings

Reply:
Updated in the comment above.